### PR TITLE
Allow GrpcWebStreamParser to accept Uint8Array

### PR DESCRIPTION
**Pull request by travikk**
Solves the following: https://github.com/grpc/grpc-web/issues/710

Suggested by: https://github.com/grpc/grpc-web/pull/683#issuecomment-588030662

### CODE REVIEW

File: javascript/net/grpc/web/grpcwebstreamparser.js [hunk 1]
```diff
@@ -175,16 +175,28 @@ Parser.prototype.error_ = function(inputBytes, pos, errorMsg) {
...
-  asserts.assert(input instanceof Array || input instanceof ArrayBuffer);
+  asserts.assert(input instanceof Array || input instanceof ArrayBuffer || input instanceof Uint8Array);
 
   var parser = this;
-  var inputBytes = (input instanceof Array) ? input : new Uint8Array(input);
+  var inputBytes;
   var pos = 0;
 
+  if (input instanceof Uint8Array || input instanceof Array) {
+    inputBytes = input
```
Comment (stanley-cheung) [inline javascript/net/grpc/web/grpcwebstreamparser.js]: nit: missing semicolon. doesn't pass closure lint checks.

File: javascript/net/grpc/web/grpcwebstreamparser.js [hunk 2]
```diff
@@ -175,16 +175,28 @@ Parser.prototype.error_ = function(inputBytes, pos, errorMsg) {
...
+  } else {
+    inputBytes = new Uint8Array(input)
```
Comment (stanley-cheung) [inline javascript/net/grpc/web/grpcwebstreamparser.js]: nit: missing semicolon. doesn't pass closure lint checks

File: javascript/net/grpc/web/grpcwebstreamparser.js (hunk 1 above)
Comment (travikk) [reply javascript/net/grpc/web/grpcwebstreamparser.js]: fixed

File: javascript/net/grpc/web/grpcwebstreamparser.js (hunk 2 above)
Comment (travikk) [reply javascript/net/grpc/web/grpcwebstreamparser.js]: fixed

File: javascript/net/grpc/web/grpcwebstreamparser.js [hunk 3]
```diff
@@ -175,16 +175,28 @@ Parser.prototype.error_ = function(inputBytes, pos, errorMsg) {
 
 
 /**
+ * Parse the new input.
+ *
+ * Note that there is no Parser state to indicate the end of a stream.
+ *
+ * @param {string|!ArrayBuffer|!Uint8Array|!Array<number>} input The input data
  * @throws {!Error} Throws an error message if the input is invalid.
- * @override
```
Comment (stanley-cheung) [inline javascript/net/grpc/web/grpcwebstreamparser.js]: Sorry another issue: apparently you need `@override` here because now the spec differs

Comment (travikk) [reply javascript/net/grpc/web/grpcwebstreamparser.js]: Fixed

**Comment by travikk** [comment]
@stanley-cheung do you think you can merge this in and release?

**Comment by stanley-cheung** [comment]
Thanks, all internal tests passed.

**Pull request merged**